src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -83,6 +83,17 @@
  * <pre>
  * &lt;module name="MethodName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   public void firstMethod1() {} // OK
+ *   protected void secondMethod2() {} // OK
+ *   private void ThirdMethod3() {} // violation, method name must match to the
```

Comment:
remove 3 from here and a space before default in the below comment.

Reply:
done.

Reply:
resolved